### DISCUSSION TITLE
NEED GEPHI HELP

### DISCUSSION THREAD

**Discussion by supermetsen** [gephi/gephi#2824] [Q&A]
I'm a newbie at Gephi. 

I have made a network with 5 youtube videoes and their users in the comments where we want to analyze with a network to discover if some of the users comment on multiple videoes. 
I have made a network in Gephi and we want to make the node sizes depending on their out-degree, the higher the out-degree, the bigger the node. 
How to do that in Gephi???

another question:
What is weighted in-degree???

thank you so much, 
Supermetsen

**Comment by niathena** [comment]
I would suggest posting questions like this to the [Gephi Facebook group](https://www.facebook.com/groups/gephi/). 

In the meantime, @seinecle's tutorial, [Simple Gephi Project from A to Z](https://seinecle.github.io/gephi-tutorials/generated-html/simple-project-from-a-to-z-en.html), is probably a good place to start. Node size can be changed in the Appearances window (using the icon to the right of the palette icon.)
